### ISSUE TITLE
RandomGenerator.getDefault().nextInt() always returns the same first value.

### ISSUE THREAD

**Issue by ogregoire** [adoptium/adoptium-support#480]
## Summary

`RandomGenerator.getDefault().nextInt()` always returns the same first value. The values afterwards seem properly random, but the first is always the same number: 715378305.

This is annoying because even if I'm doing proper usage, such as using one `RandomGenerator` for a whole app and keep using it for the rest of the lifetime, the first value is *always* the same accross all the runs of the application.

## Steps to reproduce

```
import java.util.*;
import java.util.random.*;
public class RandomGeneratorBroken {
  public static void main(String[] args) {
    var numbers = new HashSet<>();
    for (int i = 0; i < 1_000_000; i++) {
      numbers.add(RandomGenerator.getDefault().nextInt());
    }
    System.out.println(numbers.size());
  }
}
```

## Expected results

A number being 1,000,000 or close to it.

## Actual results

1

### Java version:

On MacOS:
```
openjdk version "17.0.2" 2022-01-18
OpenJDK Runtime Environment Temurin-17.0.2+8 (build 17.0.2+8)
OpenJDK 64-Bit Server VM Temurin-17.0.2+8 (build 17.0.2+8, mixed mode)
```

On Ubuntu Linux:
```
openjdk version "17.0.2" 2022-01-18
OpenJDK Runtime Environment Temurin-17.0.2+8 (build 17.0.2+8)
OpenJDK 64-Bit Server VM Temurin-17.0.2+8 (build 17.0.2+8, mixed mode, shared)
```

### What is your operating system and platform?

Tested on:
* MacOS Monterey 12.3 with M1 chip.
* Ubuntu Linux 20.04 (x86_64)

### How did you install Java?

Through IntelliJ's JDK setup

### Did it work before?

I don't know.

### Did you test with other Java versions?

No since Java 17 is the first where `RandomGenerator` is introduced.

**Comment by jerboaa** [comment]
I am not able to reproduce this with the latest JDK head (changed the reproducer to use loop bound of `100` instead of a million). It's reproducible with JDK 18 GA. It appears to be that a backport of https://bugs.openjdk.java.net/browse/JDK-8282551 might fix it:

```
$ java -showversion RandomGeneratorBroken 
openjdk version "19-internal" 2022-09-20
OpenJDK Runtime Environment (build 19-internal-adhoc.sgehwolf.jdk-jdk)
OpenJDK 64-Bit Server VM (build 19-internal-adhoc.sgehwolf.jdk-jdk, mixed mode, sharing)
100
$ java -showversion RandomGeneratorBroken
openjdk version "18" 2022-03-22
OpenJDK Runtime Environment 22.3 (build 18+37)
OpenJDK 64-Bit Server VM 22.3 (build 18+37, mixed mode, sharing)
1
```

**Comment by jerboaa** [comment]
Confirmed. I'll handle the backport.

**Comment by jerboaa** [comment]
Should be fixed once OpenJDK `17.0.4 ` gets released (July 2022)

**Comment by formingform** [comment]
I get the same issue
OS: windows 11
JDK: java version "17.0.11" 2024-04-16 LTS
```
RandomGenerator.getDefault().nextInt();
```
it always return: 715378305
